src/pymedphys/dcm/_level1/dcmdose.py

```diff
@@ -41,12 +40,32 @@
 from ...libutils import get_imports
 IMPORTS = get_imports(globals())
 
-Coords = namedtuple('Coords', 'x y z')
-Dose = namedtuple(
-    'Dose', 'values units type summation heterogeneity_correction')
-
 # pylint: disable=C0103
 
+class DicomDose:
+
+    def __init__(self, dcm_filepath):
+        dcm = pydicom.dcmread(dcm_filepath)
```

Comment:
One preference on my end, have it by default be initialised from a pydicom object, not file path. But then include a `from_filepath` `@classmethod`

This would mean the following two methods would be possible to create this object:

```
dicom_dose = DicomDose(pydicom_object)
dicom_doss = DicomDose.from_filepath(dicom_filepath_string)
```

Reply:
Fair enough. I forgot to mention it but I did think it was worth supporting both.

I'm not sure I see a scenario where you wouldn't call `DicomDose()` from a file. It seems unnecessary to force to user to perform a `pydicom.dcmread()` first, when the entire `dcm` is stored within the `DoseObject` anyway.

Reply:
When using it within our own library we might want to pass it around. As a preference thing I sort of picture it as extending an object that already exists, so it feels like it should compose with that original object :).

Reply:
I'm don't think I understand your first point re: passing the object around, but I don't really have a preference. 

Do you envisage a situation where a DicomDose object could be constructed from a pydicom Dataset that did not come from a file? If so, I think your suggestion to construct from a pydicom Dataset (rather than a file) makes sense.

Reply:
One case off the top of my head, it would be nice to be able to extend "dcm from dict" to be able to have a bumpy array within the pixel array dictionary field. Then we could make a DicomDose object from a dictionary.